File: app/src/main/java/com/developer/valyutaapp/ui/sort/SortViewInterface.java
package com.developer.valyutaapp.ui.sort;

import com.developer.valyutaapp.model.Valute;

import java.util.List;

public interface SortViewInterface {
    void showToast(String s);
    void showProgressBar();
    void hideProgressBar();
    void displayValutes(List<Valute> valute);
    void displayError(String s);
}

File: app/src/main/java/com/developer/valyutaapp/ui/sort/SortViewInterface.kt
package com.developer.valyutaapp.ui.sort

import com.developer.valyutaapp.domain.entities.Valute

interface SortViewInterface {
    fun showToast(s: String)
    fun showProgressBar()
    fun hideProgressBar()
    fun displayValutes(valute: List<Valute>)
    fun displayError(s: String)
}